lib/deployinator/views/stats.rb

```diff
@@ -0,0 +1,101 @@
+require 'json'
+require 'time'
+require './views/etsy-layout'
+
+module Deployinator::Views
+  class Stats < EtsyLayout
+
+    @@get_these_stacks = ['web', 'photos', 'blog', 'search', 'xsearch', 'supergrep', 'graphite' ]
+    @@ignore_these_stacks = ['webs','LOG MESSAGE', 'deploy', 'api', 'imagestorage' , 'storque', 'atlas' ]
+
+    def deploys
+      @deploys ||= begin
+        log_to_hash({
+          :no_global => false,
+          :stack => @@get_these_stacks,
+          :env => "production|search|prod",
+          :extragrep => "Production deploy",
+          :no_limit => true,
+          :limit => 10000
+        })
+       end
+        # note that the stack param will help but will send bring back extra lines that matched
+    end
+
+    def configs
```

Comment:
This would also need to be moved internal because it doesn't apply to the general populus